modules/index/src/integrationTest/java/org/apache/ignite/internal/index/ItRwTransactionAndIndexesTest.java

```diff
@@ -62,16 +66,23 @@ void tearDown() {
         CLUSTER.runningNodes().forEach(IgniteImpl::stopDroppingMessages);
     }
 
+    @SuppressWarnings("resource")
     @Test
-    void testCreateIndexInsideRwTransaction() {
+    void testCreateIndexInsideRwTransaction() throws Exception {
+        CatalogManager catalogManager = node().catalogManager();
+
         TableImpl table = (TableImpl) createZoneAndTable(ZONE_NAME, TABLE_NAME, 1, 1, ENGINE_NAME);
 
-        setAwaitIndexAvailability(false);
         dropAnyBuildIndexMessages();
 
         Transaction rwTx = beginRwTransaction();
 
-        createIndex(TABLE_NAME, INDEX_NAME, COLUMN_NAME);
+        runAsync(() -> createIndex(TABLE_NAME, INDEX_NAME, COLUMN_NAME));
```

Comment:
Same

Reply:
This test is built in such a way that the iindex never becomes AVAILABLE